airflow/providers/amazon/aws/sensors/rds.py

```diff
@@ -51,6 +50,12 @@ def _describe_item(self, item_type: str, item_name: str) -> list:
         elif item_type == 'export_task':
             exports = self.hook.conn.describe_export_tasks(ExportTaskIdentifier=item_name)
             return exports['ExportTasks']
+        elif item_type == "db_instance":
+            instances = self.hook.conn.describe_db_instances(DBInstanceIdentifier=item_name)
+            return instances["DBInstances"]
+        elif item_type == "db_cluster":
```

Comment:
Added `db_cluster` in https://github.com/apache/airflow/pull/26003/commits/0d537b1bdf1fa3dbd8a94f1b635a5814e84e11e9